pkgs/applications/system/qjournalctl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ 
+  stdenv,
+  fetchFromGitHub,
+  qtbase,
+  qmake,
+  pkgconfig,
+  libssh,
+	wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qjournalctl";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "pentix";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "08qfcf5yrfzsis9gkawkhshwmmi0jqw8xv23925hqass4a2xz704";
+  };
+
+  postPatch = ''
+    substituteInPlace qjournalctl.pro --replace /usr/ $out/
+  '';
+
+  nativeBuildInputs = [ qmake pkgconfig libssh ];
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.

Reply:
`libssh` belongs to `buildInputs`.